packages/nodes-base/nodes/Salesforce/__test__/node/users.workflow.json

```diff
@@ -0,0 +1,349 @@
+{
+	"name": "users.workflow",
+	"nodes": [
+		{
+			"parameters": {},
+			"type": "n8n-nodes-base.manualTrigger",
+			"typeVersion": 1,
+			"position": [0, 100],
+			"id": "c7971ec9-8bbf-42c1-b99b-dc24de009760",
+			"name": "When clicking ‘Test workflow’"
+		},
+		{
+			"parameters": {
+				"resource": "user",
+				"operation": "getAll",
+				"returnAll": true,
+				"options": {}
+			},
+			"type": "n8n-nodes-base.salesforce",
+			"typeVersion": 1,
+			"position": [220, 0],
+			"id": "b6762525-8485-4ac7-9cab-bad56c627187",
+			"name": "Salesforce",
+			"credentials": {
+				"salesforceOAuth2Api": {
+					"id": "gBXFg0tv79sQS6pc",
+					"name": "Salesforce account"
+				}
+			}
+		},
+		{
+			"parameters": {},
+			"type": "n8n-nodes-base.noOp",
+			"typeVersion": 1,
+			"position": [440, 0],
+			"id": "006a7764-e287-437a-9475-d8a78e972172",
+			"name": "No Operation, do nothing"
+		},
+		{
+			"parameters": {
+				"resource": "user",
+				"userId": "id1"
+			},
+			"type": "n8n-nodes-base.salesforce",
+			"typeVersion": 1,
+			"position": [220, 200],
+			"id": "19f7ad1c-0e32-4bf3-a12f-80dace398586",
+			"name": "Salesforce1",
+			"credentials": {
+				"salesforceOAuth2Api": {
+					"id": "gBXFg0tv79sQS6pc",
+					"name": "Salesforce account"
+				}
+			}
+		},
+		{
+			"parameters": {},
+			"type": "n8n-nodes-base.noOp",
+			"typeVersion": 1,
+			"position": [440, 200],
+			"id": "6c19df43-d32e-47ff-aecc-e045657c02c2",
+			"name": "No Operation, do nothing1"
```

Comment:
I don't think you need these noop nodes in your test workflows, you can pin the Salesforce node itself